cognite/client/data_classes/workflows.py

```diff
@@ -1175,3 +1175,185 @@ def load(cls, resource: Any, cognite_client: CogniteClient | None = None) -> Wor
 
     def dump(self, camel_case: bool = True, as_external_id: bool = False) -> list[dict[str, Any]]:
         return [workflow_id.dump(camel_case, as_external_id_key=as_external_id) for workflow_id in self.data]
+
+
+WorkflowTriggerType: TypeAlias = Literal["schedule"]
+
+
+class WorkflowScheduledTriggerRule(CogniteObject):
+    """
+    This class represents a scheduled trigger rule.
+
+    Args:
+        cron_expression(str | None): The cron specification for the scheduled trigger.
+    """
+
+    def __init__(self, cron_expression: str | None = None) -> None:
+        self.trigger_type: WorkflowTriggerType = "schedule"
+        self.cron_expression = cron_expression
+
+
+class WorkflowTriggerCreate(CogniteResource):
+    """
+    This class represents a workflow trigger for creation.
+
+    Args:
+        external_id (str): The external ID provided by the client. Must be unique for the resource type.
+        trigger_rule (WorkflowScheduledTriggerRule): The trigger rule of the workflow version trigger.
+        workflow_external_id (str): The external ID of the workflow.
+        workflow_version (str): The version of the workflow.
+        input (dict | None): The input data of the workflow version trigger. Defaults to None.
+    """
+
+    def __init__(
+        self,
+        external_id: str,
+        trigger_rule: WorkflowScheduledTriggerRule,
+        workflow_external_id: str,
+        workflow_version: str,
+        input: dict | None = None,
```

Comment:
In the spec, `input` is listed as required. I assume that is a mistake and it should be optional?

Reply:
It was a mistake indeed. We have updated the API spec and the server side too.